static/index.html

```diff
@@ -237,17 +249,20 @@ <h3>Last error</h3>
           >
         </div>
 
-        <button @click="saveForm" class="btn btn-primary">
-          Save
-        </button>
-
-        <button
-          @click="resetForm"
-          class="btn btn-outline-primary"
-          id="reset-config"
-        >
-          Reset
-        </button>
+        <div class="custom-control custom-switch mb-5">
+          <input
+            type="checkbox"
+            class="custom-control-input"
+            id="cookie-txn-switch"
+            v-model="useCookiesForTransactions"
+          />
+          <label
+            for="cookie-txn-switch"
+            class="custom-control-label"
+            data-cy="switch-cookie-txns"
+            >Use cookies for transactions</label
+          >
+        </div>
```

Comment:
Adds a switch for transaction cookies so that we can play around with it (and use it for integration testing).